ex/loqui/lib/loqui/cowboy_protocol.ex

```diff
@@ -165,20 +209,21 @@ defmodule Loqui.CowboyProtocol do
 
   @spec handle_request(tuple, state) :: {:ok, state} | {:shutdown, atom}
   defp handle_request({:hello, _flags, version, encodings, compressions}, %{ping_interval: ping_interval, supported_encodings: supported_encodings, supported_compressions: supported_compressions}=state) do
-    encoding = choose_encoding(supported_encodings, encodings)
-    compression = choose_compression(supported_compressions, compressions)
-
+    codec = choose_encoding(supported_encodings, encodings)
```

Comment:
[nit] Might want to change everything to be called `codec` for consistency.

Reply:
Agreed, but I think these are actually encodings, as they're just strings sent from the client. I'll update the struct's `:encoding` and `:compression` fields as well as the name of this function to `choose_codec` and `choose_compressor`.